src/e2e/pages/dashBoardPage.ts

```diff
@@ -210,6 +211,6 @@ export class DashboardPage {
 
   async goToFAQs() {
     await this.faqsPageLink.click();
-    await this.page.waitForURL("**/firefox-monitor-faq");
+    await this.page.waitForURL(/.*firefox-monitor-faq.*/);
```

Comment:
Yeah, I thought that it would expect the string to end with 'faq', but the URL has a redirect parameter '...=firefox-monitor-faq&redirectlocale=en-US'